src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -352,18 +390,24 @@ def test_WINNF_FT_S_FAD_1(self, config_filename):
         self.assertEqual(ppa_record['id'].split("/")[0], 'zone')
         self.assertEqual(ppa_record['id'].split("/")[1], 'ppa')
         self.assertEqual(ppa_record['id'].split("/")[2], self._sas._sas_admin_id)
-        del ppa_record['id']
-        # TODO : to update the following check according when PAL grants are supported in this test case
-        if 'cbsdReferenceId' in ppa_record['ppaInfo']:
-          self.assertFalse(ppa_record['ppaInfo']['cbsdReferenceId'])
-          del ppa_record['ppaInfo']['cbsdReferenceId']
+        del ppa_record['id'] 
         # verify that the injected ppas exist in the dump files
-        # TODO: add Check for GeoJSON Winding of ppa polygon  
+        # check GeoJson Winding of PPA record
+        utils.HasCorrectGeoJsonWinding(ppa_record['zone']['features'][0]['geometry'])     
         exist_in_dump = False
         for ppa in config['ppaRecords']:
           if 'id' in ppa:
-            del ppa['id']        
-          exist_in_dump = exist_in_dump or areTwoPpasEqual(ppa_record, ppa)
+            del ppa['id'] 
+            # we exclude the values of the CBSD reference Id, because they are not necessarily the same
+            ppa_cbsd_reference_ids = ppa['cbsdReferenceId']
```

Comment:
This should be un-indented once, right? Otherwise this code will not work if you have more than one PPA.

Reply:
thnx, fixed.